cmd/podman/pods/inspect.go

```diff
@@ -27,6 +27,9 @@ var (
 )
 
 func init() {
+	if !registry.IsRemote() {
+		inspectCmd.Example += "\n  podman pod inspect --latest"
+	}
```

Comment:
Do you need a containerInspectCommand here too?  There are a few other commands that only have one line in this if, instead of 2 like the majority do.

Reply:
These are pod commands not container commands.  Only containers commands can be called in two ways

podman COMMAND
podman container COMMAND

There is no equivalent for Pods, you always need to specify
podman pod COMMAND>